### ISSUE TITLE
componentSeparatorColor doesnt' work

### ISSUE THREAD

**Issue by skywinder** [maxkonovalov/MKDropdownMenu#3]
```
dropdown.rowSeparatorColor = UIColor.blackColor()
dropdown.componentSeparatorColor = UIColor.blackColor()
```

 does not change color of components.
As you can see on screenshot , only first row changes color to black.

![screen shot 2016-07-19 at 17 57 36](https://cloud.githubusercontent.com/assets/3356474/16954651/5128b714-4dda-11e6-9dd2-2aa6baaf1961.png)

**Comment by maxkonovalov** [comment]
Hey @skywinder!
I tried to reproduce your issue in the example project, but all separator colors are changed as expected there, so could you please provide some more details on your setup.

Also please note that `componentSeparatorColor` is for the vertical lines dividing the header components, and `rowSeparatorColor` is for the horizontal separators.

**Comment by skywinder** [comment]
@maxkonovalov sadly, I'm quite limit in time now.
But I fixed it by changing:

```
-    self.view.backgroundColor = [UIColor colorWithWhite:0.0 alpha:kDefaultBackgroundDimmingOpacity];
+    self.view.backgroundColor = [UIColor blackColor];
...
-    mask.fillColor = [[UIColor whiteColor] CGColor];
+    mask.fillColor = [[UIColor blackColor] CGColor];
...
-    self.tableView.backgroundColor = [UIColor whiteColor];
+    self.tableView.backgroundColor = [UIColor blackColor];

```

My Base initialisation of your dropdown looks like this (it's init in subclass from your Dropdown menu, that I call :

```
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        fxInit()
    }

    func fxInit() {
        let color = ThemeColors.Red.color()
        self.dropdownBackgroundColor = color
        self.disclosureIndicatorImage = UIImage(named: "arrow_down")
        self.rowSeparatorColor = UIColor.blackColor()
        self.componentSeparatorColor = UIColor.blackColor()

        self.componentTextAlignment = NSTextAlignment.Left
        self.backgroundColor = color
    }
```

**Comment by maxkonovalov** [comment]
@skywinder looks like you have a subclass of the dropdown menu and set it up in the `init()`, which is not the intended use-case. The MKDropdownMenu instantiates a view controller and some additional classes inside its `init`, so it is not recommended to make any appearance changes until it is fully set up. 
If you create the menu in code, I'd recommend to adjust the properties after the `MKDropdownMenu`'s `-init`. And for storyboard/xib setup, just set it up in the `-viewDidLoad` of your containing view controller. See the Example project for more info.

**Comment by skywinder** [comment]
@maxkonovalov in this case I have to manually setup each time dropown menu. 
As I can see, I start my persinal init only aftere you init process is finished. Am I Wrong?

the second init methis looks like this:

```
 override init(frame: CGRect) {
        super.init(frame: frame)
        fxInit()
    }
```

BTW: 
I tried to init in in `viewDidload`, but have no luck. same results.

**Comment by maxkonovalov** [comment]
@skywinder ok, I see your point. I haven't tried this control with swift, but my guess is that even though you call your `fxInit` after `super.init`, the inner view controller's `viewDidLoad` might not get called yet by that time, and so the dropdown is not fully set up and ready to be customized. 
I will investigate this issue further and will let you know of the workaround.

**Comment by skywinder** [comment]
Ok. Thank you. Get in touch.
But it's strange, since not initializers except this 2 is called. And i setup view after them.

**Comment by maxkonovalov** [comment]
Hey @skywinder, can you please check the [**appearance**](https://github.com/maxkonovalov/MKDropdownMenu/tree/appearance) branch - I added support for `UIAppearance` for most properties, so seems like you can do without subclassing here. Anyway, your issue with `rowSeparatorColor` not setting correctly in `init` has also been fixed in this branch, I checked and it worked as expected for views created both in code and in Interface Builder.

If you confirm that you don't have any issues with this, I'll merge these changes and release a new version.

**Comment by skywinder** [comment]
Hi! Seems it works thanks! :)

**Comment by maxkonovalov** [comment]
@skywinder great! Thanks for your feedback. I released these changes in version [1.2](https://github.com/maxkonovalov/MKDropdownMenu/releases/tag/1.2), you can run `pod update` to get it.
